core/arch/arm/plat-imx/drivers/imx_scu.c

```diff
@@ -24,9 +30,8 @@ static TEE_Result scu_init(void)
 	io_write32(scu_base + SCU_NSAC, SCU_NSAC_CTRL_INIT);
 
 	/* SCU enable */
-	io_write32(scu_base + SCU_CTRL, io_read32(scu_base + SCU_CTRL) | 0x1);
+	io_setbits32(scu_base + SCU_CTRL, BIT32(0))
```

Comment:
Unrelated change, should be a separate patch.

Reply:
I will remove that chage